robocorp-code/src/robocorp_code/robocorp_language_server.py

```diff
@@ -990,55 +1015,41 @@ def _get_locators_json_info(
             }
 
         path = Path(params["robotYaml"])
-        locators_json = path.parent / "locators.json"
         locators_json_info: List[LocatorEntryInfoDict] = []
         locator: Locator
-        if locators_json.exists():
+        db, locators_json = None, None
+        try:
+            action_result: ActionResultDictLocatorsJson = self._load_locators_db(path)
+            if action_result["success"]:
+                db, locators_json = action_result["result"]
+            else:
+                raise Exception(action_result["message"])
```

Comment:
This seems a bit odd for me... 

i.e.: Since `ActionResultDictLocatorsJsonInfo` has a value indicating the success, I think you should just return one with a `success: False` (otherwise it could just return a `List[LocatorEntryInfoDict]` and throw exceptions).

Reply:
Modified the if-else to return rather than raise Exception